test/puppetlabs/puppetdb/pdb_routing_test.clj

```diff
@@ -52,7 +48,7 @@
 
       (let [resp (export svc-utils/*base-url*)]
         (tu/assert-success! resp)
-        (is (.contains (get-in resp [:headers "Content-Disposition"]) "puppetdb-export"))
+        (is (.contains (get-in resp [:headers "content-disposition"]) "puppetdb-export"))
```

Comment:
what happened here? are we returning the lowercased verison? or is the client lib doing that 'for' us now? The uppercased one is the real name..